src/runtime_config/operator_gov.lua

```diff
@@ -0,0 +1,130 @@
+-- Copyright (c) Microsoft Corporation. All rights reserved.
+-- Licensed under the Apache 2.0 License.
+
+-- This file defines the default initial contents (ie, Lua scripts) of the gov_scipts table.
+return {  
+  pass = [[
+  tables, calls, votes = ...
+
+  -- defines which of the members are operators
+  function is_operator(member)
+    return member == "0"
+  end
+
+  -- defines calls that can be passed with sole operator input
+  operator_calls = {
+    accept_node=true,
+    retire_node=true,
+    new_code=true
+  }
+
+  operator_votes = 0
+  member_votes = 0
+
+  for member, vote in pairs(votes) do
+    if vote then
+      if is_operator(member) then
+        operator_votes = operator_votes + 1
+      else
+        member_votes = member_votes + 1
+      end
+    end
+  end
+
+  -- count active members, excluding operators
+  members_active = 0
+  STATE_ACTIVE = 1
+
+  tables["ccf.members"]:foreach(function(member, details) 
+    if details["status"] == STATE_ACTIVE and not is_operator(tostring(member)) then
+      members_active = members_active + 1 
+    end 
+  end)
+
+  -- check for raw_puts to sensitive tables
+  SENSITIVE_TABLES = {"ccf.whitelists", "ccf.gov_scripts"}
```

Comment:
We should add some tests for writing on sensitive tables at some point (outside of this PR).

Reply:
We have one at line 788